hank-client/src/test/java/com/liveramp/hank/config/yaml/TestYamlHankSmartClientConfigurator.java

```diff
@@ -42,6 +45,8 @@ public void testIt() throws Exception {
     pw.println("  establish_connection_timeout_ms: 100");
     pw.println("  query_timeout_ms: 42");
     pw.println("  bulk_query_timeout_ms: 142");
+    pw.println("  preferred_environment_key: PATH");
```

Comment:
haha it was the only variable I could find which resolved when running in intellij